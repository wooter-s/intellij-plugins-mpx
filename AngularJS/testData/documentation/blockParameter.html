@if(foo; a<caret>s bar) {

}