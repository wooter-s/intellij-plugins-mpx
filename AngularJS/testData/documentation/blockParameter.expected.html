<div class='definition'><pre><span style='color: #000080'>parameter</span> <span style='color: #000000'>as</span></pre></div>
<div class='content'>
<p>Use <code>as</code> parameter to reference conditional expression results:</p><pre><code class="language-Angular2Html">@if (users$ | async; as users) {
  {{ users.length }}
}
</pre>
</div><table class='sections'>
<tr><td valign='top' class='section'><p>Library:</td><td valign='top'><p>@angular/core@17.0.0-rc.0</td>
</table>